internal/karma/karma.conf.js

```diff
@@ -117,7 +117,7 @@ var requireConfigContent = `
   var allFiles = [TMPL_user_files];
   var allTestFiles = [];
   allFiles.forEach(function (file) {
-    if (/(spec|test)\\.js$/i.test(file) && !/\\/node_modules\\//.test(file)) {
+    if (/[^a-zA-Z0-9](spec|test)\\.js$/i.test(file) && !/\\/node_modules\\//.test(file)) {
```

Comment:
add word separator to filter (same as `jasmine_node_test`)

Reply:
either way okay with me, this is close to landing